docs/products/mysql/howto/migrate-db-to-aiven-via-console.rst

```diff
@@ -0,0 +1,247 @@
+Migrate MySQL® databases to Aiven using the console 
+===================================================
+
+You can migrate MySQL® databases to Aiven using either :doc:`CLI </docs/products/mysql/howto/migrate-from-external-mysql>` or the `Aiven web console <https://console.aiven.io/>`_. This article addresses the latter scenario by providing guidelines on how to use the Aiven console to migrate MySQL databases to the Aiven platform.
+
+.. seealso::
+
+    For the other migration method (CLI), see :doc:`Migrate to Aiven for MySQL from an external MySQL </docs/products/mysql/howto/migrate-from-external-mysql>`.
+
+About migrating via console
+---------------------------
+
+* Console migration tool enables you to migrate MySQL databases to managed MySQL clusters in your Aiven account. You can migrate
+
+  * Existing on-premise MySQL databases
+  * Cloud-hosted MySQL databases
+  * Managed MySQL database clusters on Aiven.
+
+* Console migration tool can use either logical replication or physical replication.
+
+  * Logical replication allows the continuous migration, which is recommended and used by default in the tool, hence, also detailed in this guide. With this method, data transfer is possible not only for the data that has already been there in the source database when triggering the migration but also for any data written to the source database during the migration.
+
+  * Physical replication allows the other migration method supported in the migration tool, ``mysqldump``. With this method, current contents of the database are copied into a file and transferred to the target database. Any changes written to the source database during the migration are not transferred. When you trigger the migration setup in the console and initial checks detect that your source database does not support the logical replication, you are notified about it via wizard. To continue with the migration, you can select the alternative ``mysqldump`` migration method in the wizard.
```

Comment:
I wouldn't call this one physical replication it's a dump/restore

Reply:
Farther down it's referred to as **one-time snapshot (dump method)**

Reply:
> 

In those two points I compare two types of replication, logical vs physical, not migration is secondary here. And dump method is the migration method, not replication type. So we need a type of replication that is supported for the dump method. Any idea? If it's not physical replication, what replication is supported for the dump method?
Also, physical/logical replication types I borrowed from DO article on the same feature (https://docs.digitalocean.com/products/databases/mysql/how-to/migrate/#logical-replication-vs-physical-replication)

Reply:
Replication in itself is continuously streaming data. Physical or logical only defines if the changes are on a block- or on a statement-level. `mysqldump` doesn't replicate at all, it exports (dumps) the database into a text-file and re-imports it (restore).

In the DO article they do have it a bit wrong as well, as replication always means that the database is being streamed, one way or another.

Reply:
@packi , many thanks for this clarification :-) Updating accordingly